src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy

```diff
@@ -60,21 +65,21 @@ class VxlanFlowSpec extends HealthCheckSpecification {
 [#data.encapsulationCreate.toString() -> #data.encapsulationUpdate.toString(), #swPair.hwSwString()]"(Map data, SwitchPair swPair) {
         when: "Create a flow with #encapsulationCreate.toString() encapsulation type"
         sleep(10000) //subsequent test fails due to traffexam. Was not able to track down the reason
-        def flow = flowHelperV2.randomFlow(swPair)
-        flow.encapsulationType = data.encapsulationCreate
-        flowHelperV2.addFlow(flow)
+        def flow = flowFactory.getBuilder(swPair)
+                .withEncapsulationType(data.encapsulationCreate)
+                .build().create()
 
         then: "Flow is created with the #encapsulationCreate.toString() encapsulation type"
-        def flowInfo = northboundV2.getFlow(flow.flowId)
-        flowInfo.encapsulationType == data.encapsulationCreate.toString().toLowerCase()
+        def flowInfo = flow.retrieveDetails()
```

Comment:
can omit this call as we are waiting for flow to be in UP state and it is the call flow.retrieveDetails()

Reply:
Done, removed